main.py

```diff
@@ -12,14 +12,16 @@
 
 """
 import logging
-# noinspection PyUnresolvedReferences
-from Modules import cli_manager
 
 from pydle import ClientPool, Client
 
+# noinspection PyUnresolvedReferences
+from Modules import cli_manager
+from Modules.rat_command import Commands
+# noinspection PyUnresolvedReferences
+from commands import debug as cmd_debug
```

Comment:
the noinspection is to prevent pycharm from removing the import when i reformat file / optimize imports. and yeah, doing something in `Module.__init__` may be a better approach

Reply:
Aight, thanks for the info